crates/kubelet/src/kubelet.rs

```diff
@@ -65,10 +65,14 @@ impl<T: 'static + Provider + Sync + Send> Kubelet<T> {
 
         // This informer listens for pod events.
         let provider = self.provider.clone();
-
+        let node_name = self.config.node_name.clone();
```

Comment:
nit: you can avoid an extra allocation if you do the formatting (with `format!`) here.

`let field_selector = format!("spec.nodeName={}", self.config.node_name)`